core/changedetector/proto_change_detector_service.go

```diff
@@ -0,0 +1,231 @@
+package changedetector
+
+import (
+	"context"
+	"errors"
+	"github.com/golang/protobuf/proto"
```

Comment:
`github.com/golang/protobuf/proto` is deprecated: Use the `google.golang.org/protobuf/proto` package instead.

Reply:
updated